### DISCUSSION TITLE
How to import custom fonts?

### DISCUSSION THREAD

**Discussion by yukibtc** [hecrj/iced#2001] [Q&A]
Hi, I'm bumping `iced` to v0.10 and seems that is no longer possible to import external font. 
Exists a way to import external fonts from `iced` v0.10? 
It can be useful to import icon fonts or others, avoiding to require the user to manually install it.
Thanks

**Comment by akavel** [comment]
I think this is now answered here: https://github.com/iced-rs/iced/discussions/1988
